src/api.ts

```diff
@@ -184,8 +184,8 @@ function createRelease(api: ApiClient, body: models.ReleaseAttributes): Promise<
 function createImage(
 	api: ApiClient,
 	release: number,
-	labels: Dict<string>,
-	envvars: Dict<string>,
+	labels: Dict<string> | undefined,
```

Comment:
Declaring this as `labels?: Dict<string>` is slightly clearer.

Reply:
It's not allowed though, since optional (`?`) fn args are only allowed when they are at the end, and in this case there is also `body`. For sure I could change the arg ordering, but I tried to not introduce any code changes not related to the typing erros.